docs/settings.md

```diff
@@ -134,6 +134,7 @@ Legend:
 | `backgrounds_branded_for_guests`     | string<br>`1` or `0`                                             | `0`        | No   |        | Whether guests are allowed to use the virtual backgrounds provided via `themes/talk-backgrounds/`                                                                                                                                                                                |
 | `backgrounds_default_for_users`      | string<br>`1` or `0`                                             | `1`        | No   |        | Whether users are allowed to use the default virtual backgrounds provided by the releases                                                                                                                                                                                        |
 | `backgrounds_upload_users`           | string<br>`1` or `0`                                             | `1`        | No   |        | Whether users are allowed to upload custom virtual backgrounds and choose from their Nextcloud Files                                                                                                                                                                             |
+| `chat_relay`                         | string<br>`1` or `0`                                             | `0`        | No   |        | Whether Talk will use the signaling backend to send chat messages                                                                                                                                                                                                                |
```

Comment:
Should be an experiment below
(for tracking and conflict avoidance the experiment doc change needs to be backported to main afterwards

Reply:
Experiments are automatically exposed as capabilities, so the clients can see the flag as well